### PR TITLE
Add unlock CLI command

### PR DESCRIPTION
**Pull request by saraycp**

### CODE REVIEW

**Review by eduardoj — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -34,17 +34,31 @@ Require the gem whenever you need it:
...
 It is enough to enable `repo:status` and `repo_deployment` scopes.
 
-From development environment:
+You can pass them using the corresponding flags as we do in this example:
+
+```
+obs_github_deployments unlock --repository $GITHUB_REPOSITORY --token $GITHUB_TOKEN
+```
+
+But you could set them as environment variables in the `.env` file taking `.env.example` as example. So you don't need to specify them in the command:
```
Comment (danidoni) [inline README.md]:
```suggestion
But you could also set them as environment variables in the `.env` file, taking `.env.example` as example. Doing so you don't need to specify them in the command:
```

**Review by danidoni — APPROVED** [review]
I left a small :thought_balloon:, other than that, this LGTM

**Comment by saraycp** [comment]
I've moved this to draft, we came to an agreement on how to handle exceptions and how the output and the returned value should be. Working on that.

File: README.md (hunk 1 above)
Comment (saraycp) [reply README.md]: done

**Review by krauselukas — APPROVED** [review]

**Pull request merged**